_posts/2016-08-18-On-Emission.markdown

```diff
@@ -0,0 +1,130 @@
+
+Interestingly if the router cannot route a view controller, it will pass through to a web view. This is why we consider the app a [hybrid app][hybrid_app]. This system means adding new view controllers is extremely easy.
+
+### Introducing Emission
+
+Emission is:
+
+- A node module.
+- A CocoaPod.
+- An iOS App.
+
+#### The Node Module
+
+Emission itself, is a node module. In our case, it is a JavaScript library that exposes 3 JavaScript objects.
+
+``` javascript
+/* @flow */
+'use strict';
+
+import Containers from './lib/containers';
+import Components from './lib/components';
+import Routes from './lib/relay/routes';
+
+import './lib/relay/config';
+import './lib/app_registry';
+
+export default {
+  Containers,
+  Components,
+  Routes,
+};
+```
+
+Another node project can have Emission as a dependency - then can access our `Container`s, `Component`s and `Route`s. A container is a [Relay container][relay_cont], a component is a [React Component][react_component] and a Route is a [Relay Route][relay_route].
+
+The thing that's interesting from the integration side, is that each `Container` is effectively a View Controller that Emission provides to a host application. React Native ignores  the concept of View Controllers from the Cocoa world, so we have an [ARComponentViewController][arcomponent] which is subclassed for each exposed `Component` class. 
+
+#### The iOS App
+
+The iOS app acts as a host target for the CocoaPod, and provides an instance of an [AREmission][ar_emission] object to the view controllers using React Native. The app is nothing special, it is the default app that is created using `pod lib create`. We then [use CocoaPods][pods_emission] to bring in React from inside the `node_modules/` folder the Emission node module creates.
+
+The `AREmission` instance is the intermediary between the host-app ([The Emission Example app][example_emission], or [Eigen][eigen_emission].) It has an API for handling routing, and passing authentication credentials into the React Native world.
+
+We use the example app to do development inside React Native. As of right now, it is simply a tableview that provides a list of View Controllers [that represent an exposed Container][app_delegate_cont]. Once you are in the right view controller, you can rely on [Hot Reloading][reloading] to simplify your work.
+
+#### The Pod
+
+An important part of working with React Native, is that you can choose to use native code when appropriate. The [Pod for][podspec] Emission, created entirely in Objective-C, provides:
+
+* Communication between React Native and the host app objects via [native modules][native_modules] 
+* `UIViewController` subclasses for Host apps to consume.
+* Bridges for native views into React Native.
```

Comment:
Maybe a line on why that's different from the communication via native modules? E.g. if the top one is only from RN to Host, and bridges are from Host to RN.